core/simulations/index.ts

```diff
@@ -0,0 +1,43 @@
+import prompts, { PromptObject } from 'prompts';
+import { SIMULATIONS } from './config';
+import readline from 'readline';
+
+const keypress = async (title: string) => {
+    await prompts({
+        type: 'invisible',
+        name: 'irrelevant',
+        message: `Press Enter to proceed to the next step "${title}"`,
+    });
+};
+
+const selectAndRunSimulation = async () => {
+    readline.emitKeypressEvents(process.stdin);
+    const promtChoices = SIMULATIONS.map(simulationConfig => simulationConfig.title).map(simulationName => ({
+        title: simulationName,
+        value: simulationName,
+    }));
+    const promtConfig: PromptObject = {
```

Comment:
According to the https://github.com/sidestream-tech/unified-auctions-ui/pull/426#discussion_r962707075
```suggestion
    const selectSimulationPrompt: PromptObject = {
```